src/ui/checkerboard.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2018 The boardgame.io Authors.
+ *
+ * Use of this source code is governed by a MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Checkerboard
+ *
+ * Component that will show a configurable checker board for games like
+ * chess, checkers and others. The vertical columns of squares are labeled
+ * with letters from a to z, while the rows are labeled with numbers, starting
+ * with 1.
+ *
+ * Props:
+ *   rows    - How many rows to show up, 8 by default.
+ *   cols    - How many columns to show up, 8 by default. Maximum is 26.
+ *   onClick - On Click Callback, (row, col) of the square passed as argument.
+ *   primaryColor - Primary color, #d18b47 by default.
+ *   secondaryColor - Secondary color, #ffce9e by default.
+ *   colorMap - Object of object having cell names as key and colors as values.
+ *   Ex: { 'c5': 'red' } colors cells c5 with red.
+ *
+ * Usage:
+ *
+ * <Checkerboard>
+ *   <Token square={'c5'}>
+ *     <Knight color='dark' />
+ *   </Token>
+ * </Checkerboard>
+ */
+class Checkerboard extends React.Component {
```

Comment:
Can we keep `Checkerboard` in the examples section? I don't think it will be useful to games beyond chess and checkers (i.e. I don't think it should be distributed).

Reply:
Sure